src/main/java/com/wecca/canoeanalysis/controllers/modules/PercentOpenAreaController.java

```diff
@@ -138,24 +139,52 @@ public void handleAnalyzeImageButton() {
     }
 
     private double getPoaFromImage(File file) {
+        Color color = colorPicker.getValue();
+        int r = (int) (color.getRed() * 255);//get the RGB color of the ColorPicker
+        int g = (int) (color.getGreen() * 255);
+        int b = (int) (color.getBlue() * 255);
+        int count = 0;//counter for counting the pixels closest to the RGB color of the ColorPicker
+
         ImagePlus img = IJ.openImage(file.getAbsolutePath());
         ImageProcessor ip = img.getProcessor();
-        if (ip.getBitDepth() != 8)
-            IJ.run(img, "8-bit", "");
-        IJ.run(img, "Subtract Background...", "rolling=50");
-        IJ.run(img, "Gaussian Blur...", "sigma=2");
-        IJ.setAutoThreshold(img, "Default");
-        IJ.run(img, "Convert to Mask", "");
-        IJ.run(img, "Set Measurements...", "area redirect=None decimal=3");
-        IJ.run(img, "Analyze Particles...", "exclude clear");
+        IJ.run(img, "RBG Color", "");
+        //IJ.run(img, "Subtract Background...", "rolling=50");
+        //IJ.run(img, "Gaussian Blur...", "sigma=2");
```

Comment:
just want to check with @Jaredsegal1 who did all these IJ.run commands that its okay to delete them, other than than looks good :)